packages/documentation/.vitepress/configuration/vite.ts

```diff
@@ -5,9 +5,6 @@ export const vite = {
   resolve: {
     alias: {
       '@': path.resolve(__dirname, '../../../ui/src'),
-      'components': path.join(__dirname, '../../components'),
-      '@demos': path.join(__dirname, '../../../examples/demos'),
```

Comment:
How then aliases will work in docs demos?

Reply:
I didn't find usages of `@ demos`